### PR TITLE
yaml standards

### PR DESCRIPTION
**Pull request by vitek-rostislav**
| Q             | A
| ------------- | ---
|Description, reason for the PR| re-implementation of https://github.com/shopsys/shopsys/pull/539 (easier than rebasing the original branch due to complicated conflicts). For testing and review purposes only.
|New feature| Yes/No/Maybe 
|[BC breaks](https://github.com/shopsys/shopsys/blob/master/docs/contributing/backward-compatibility-promise.md)| Yes/No/Maybe 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes/No/Maybe

### CODE REVIEW

**Comment by boris-brtan** [comment]
looks like we need to use the same paths as for ecs 

            <arg path="./packages/*/src"/>
            <arg path="./packages/*/tests" />
            <arg path="./packages/*/*.md"/>
            <arg path="./*.md"/>
            <arg path="./docs"/>

except mds will be yml or ym?l

**Comment by boris-brtan** [comment]
ooh daaamn, travis checker stopped to work after the fixer result.

**Comment by sspooky13** [comment]
hello @vitek-rostislav,
try run travis again. Now it should work 🙂
BTW: don't forget run `composer update` 😊

**Comment by vitek-rostislav** [comment]
Hello Mr. @sspooky13, thanks for the quick fix! Not it works better definitely! :slightly_smiling_face: 

I still get an error when the skipped file does not exist (I want to skip `docker-sync.yml` in our project, however, I do not use this file locally at all) - but I can live with that :smile: 
```
Error was caught: RecursiveDirectoryIterator::__construct(/var/www/html/docker-sync.yml): failed to open dir: No such file or directory
```

**Comment by sspooky13** [comment]
omg 😀 I forgot on option that the file not need to exist 😀 well, again next time 😄

File: docker/conf/docker-sync.yml.dist [hunk 1]
```diff
@@ -3,25 +3,25 @@ syncs:
...
-        ]
+        sync_excludes:
+            - .ci
+            - .git
+            - .github
+            - .idea
+            - .docker-sync
+            - .DS_Store
+            - docker
+            - microservices
```
Comment (boris-brtan) [inline docker/conf/docker-sync.yml.dist]: there are no microservices anymore

Comment (vitek-rostislav) [reply docker/conf/docker-sync.yml.dist]: well, this should be in a separate PR :)

Comment (vitek-rostislav) [reply docker/conf/docker-sync.yml.dist]: :+1: fixed

**Comment by vitek-rostislav** [comment]
hopefully will be finshed in the original PR (#539)

**Pull request closed without merging**